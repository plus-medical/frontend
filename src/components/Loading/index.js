import React from 'react'
import LoadingGif from '../../assets/images/loading.gif'
import './styles.scss'
function Loading () {
  return (
    <div className='loading'>
      <img src={LoadingGif} alt='Cargando' />
    </div>
  )
}
export default Loading
