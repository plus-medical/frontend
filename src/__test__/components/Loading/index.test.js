/* eslint-disable no-undef */
import React from 'react'
import { create } from 'react-test-renderer'
import RouterMock from '../../../__mocks__/RoutesMocks'
import Loading from '../../../components/Loading/index'

describe('<Loading />', () => {
  test('Review create in Loading component', () => {
    const loading = create(
      <RouterMock>
        <Loading />
      </RouterMock>
    )
    expect(loading.toJSON()).toMatchSnapshot()
  })
})
