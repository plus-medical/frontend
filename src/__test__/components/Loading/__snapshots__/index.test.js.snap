// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Review create in Loading component 1`] = `
<div
  className="loading"
>
  <img
    alt="Cargando"
    src="file-stub"
  />
</div>
`;
